portal/src/main/webapp/js/src/cross-cancer-plotly-plots.js

```diff
@@ -528,10 +528,12 @@ var ccPlots = (function (Plotly, _, $) {
 
     return {
         init: function() {
-            var tmp = setInterval(function () {timer();}, 1000);
```

Comment:
@alisman should we just remove this function completely?

Reply:
i'm inclined to just leave stuff and then it will get cleaned out when we go SPA